brave/brave-core
# Rewards category mojo

Resolves https://github.com/brave/brave-browser/issues/5411

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- claim grant
- tip to unverified publisher
- make sure that you see tip in pending modal and that category is correct
- make recurring tip (and make sure that is correctly displayed in settings page)
- make tip to verified publisher and make sure that is recorded in wallet section / in tip section

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
